src/test/java/duke/task/TaskListTest.java

```diff
@@ -0,0 +1,100 @@
+package duke.task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import duke.DukeException;
+
+class TaskListTest {
```

Comment:
You can consider adding class and method header comments for your various test classes to help developers understand what exactly you are testing for in each test case / test class